### DISCUSSION TITLE
Date picker not appearing for a input parameter

### DISCUSSION THREAD

**Discussion by harshitambasht1** [eclipse-birt/birt#1731] [Q&A] [answered]
BIRT version -- 4.12
jdk version - 11
We have developed the report on birt which has two input parameters of type date. When we execute the report the date picker is not appearing. And when we view the source, for these two input parameters the type set to TEXT instead of date. Please anyone can help to resolve this issue.
![Screenshot from 2024-06-14 12-19-27](https://github.com/eclipse-birt/birt/assets/113333970/668812ef-b8f8-48ee-8926-a6bd79f34fde)

**Comment by speckyspooky** [comment, marked as answer]
Yes of course, the option with the datepicker is available with BIRT 4.14 - see  #1356
